src/PhysicalModels/MechanicalModels.jl

```diff
@@ -95,67 +95,110 @@ struct EnergyInterpolationScheme{M1<:Mechano,M2<:Mechano} <: PhysicalModel
 end
 
 
-struct ComposedElasticModel <: Elasto
-  Model1::Elasto
-  Model2::Elasto
+struct MultiElastic{A,B} <: Elasto
```

Comment:
After the conversations, I propose to name it as `ComposedIsoElastic`

There is an idiomatic reason:
- Composed -> `+`
- Multi -> `Tuple`